src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx

```diff
@@ -121,8 +126,51 @@ class CreateSchemeRegistrarProposal extends React.Component<IProps, IState> {
     this.setState({tags});
   }
 
+  public exportFormValues(values: IFormValues) {
+    values = {
+      ...values,
+      ...this.state,
+    };
+    exportUrl(values);
+    this.props.showNotification(NotificationStatus.Success, "Exportable url is now in clipboard :)");
+  }
+  
+  loadCurrentTab(){
+    let initialCurrentTab = importUrlValues({ schemeToAdd: "", schemeToEdit: "", schemeToRemove: ""});
+    if(initialCurrentTab.schemeToAdd) {
+      initialCurrentTab = "addScheme";
+    }
+    else if(initialCurrentTab.schemeToEdit) {
+      initialCurrentTab = "editScheme";
+    }
+    else if(initialCurrentTab.schemeToRemove) {
+      initialCurrentTab = "removeScheme";
+    }
+    else {
+      initialCurrentTab = "addScheme";
+    }
+
+    return initialCurrentTab;
+  }
+  
   public render(): RenderOutput {
     // "schemes" are the schemes registered in this DAO
+    const initialFormValues: any = {
```

Comment:
again seems better to create this and call the import in the constructor so it only happens once